core/helpers/EEH_Sideloader.helper.php

```diff
@@ -14,15 +14,47 @@
 class EEH_Sideloader extends EEH_Base
 {
 
+    /**
+     * @since   4.1.0
+     * @var     string
+     * @access  private
+     */
```

Comment:
awesome thnx for adding this 👍 

we don't need the `@access` tag however since `private` is clearly added before the property
any modern IDE will properly detect the property scope without the need of this tag, so this can be removed